src/BenchmarksApps/TodosApi/DataExtensions.cs

```diff
@@ -0,0 +1,192 @@
+﻿using System.Data;
+
+namespace Npgsql;
+
+internal static class DataExtensions
+{
+    public static async Task<int> ExecuteAsync(this NpgsqlDataSource dataSource, string commandText)
```

Comment:
Any particular reason `CancellationToken` isn't passed anywhere here?

Reply:
Not really, just 'cos I'm lazy. It's good to add them though.

Reply:
Yeah, passing `CancellationToken` can be quite a pain

Reply:
I updated it to accept and pass `CancellationToken`